rebar.config

```diff
@@ -1,6 +1,10 @@
+%% -*- mode:erlang -*-
 {minimum_otp_vsn, "21.0"}.
 
-{deps, [{jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.5"}}}]}.
+{deps, [ {jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.5"}}}
+       , {snabbkaffe, {git, "https://github.com/kafka4beam/snabbkaffe", {tag, "0.8.2"}}}
+       , {gen_rpc, {git, "https://github.com/zmstone/gen_rpc", {branch, "fix-deprecated-api-warning"}}}
```

Comment:
is this from my commits ?
emqx/gen_rpc 2.5.0 (the version used in emqx.git) actually has the deprecated APIs fixed

Reply:
Thanks, I didn't know about the fork